zerver/tests/test_subs.py

```diff
@@ -842,11 +863,11 @@ def test_subscriptions_add_too_long(self):
 
     def test_user_settings_for_adding_streams(self):
         with stub(UserProfile, 'can_create_streams', lambda self: True):
-            result = self.common_subscribe_to_streams(self.test_email, ['stream1'])
+            result = self.common_subscribe_to_streams(self.test_email, ['usersettingsaddstreams1'])
             self.assert_json_success(result)
 
         with stub(UserProfile, 'can_create_streams', lambda self: False):
-            result = self.common_subscribe_to_streams(self.test_email, ['stream1'])
+            result = self.common_subscribe_to_streams(self.test_email, ['usersettingsaddstreams2'])
```

Comment:
I believe stream1 already exists on the testing server, so this wouldn't test anything on the best case.